### ISSUE TITLE
Worlds have no easy way to show recently updated in the World Browser 

### ISSUE THREAD

**Issue by TehTurk** [Neos-Metaverse/NeosPublic#3204]
### Is your feature request related to a problem? Please describe.

Currently in the world browser, in terms of priority worlds are shown either by latest Publish Date or by the number of users that may be in the world.

This is a problem because that means the system right now favors fresher maps, instead of users being able to update older maps, or encouraging users to work on new maps vs refining experiences. This has come up recently in a couple of conversations with some world makers.

### Relevant issues

#3104 #204 Possibly? But different in issue scope

### Describe the solution you'd like

Ideally, allow either for recently updated to be a part of the publish Category, or for it to have its own category as well too. But I also understand that the world browser categories are getting a bit cramped too.

### Describe alternatives you've considered

There are not really any.

### Additional context

This came from a discussion on Map building in Neos, and how sometimes more content discovery of newer more published worlds is favored over more refined content.

**Comment by Frooxius** [comment]
This is already supported on technical level. If you go to the "Custom" and set the sorting by "Modified Date", this will sort by last modified.

We could add some "recently updated category" too, or perhaps even consider switching the defaults for the main categories and instead having some sort of "New" field. Question is what would be best way to go about this though, we don't want to double every single category. 

I'd probably add some new UI element to allow easy toggling between "Updated" vs "New" sorting. Maybe even "Popular". That might make it easier to select in any category.

**Comment by TehTurk** [comment]
Yeah ^^, I was aware of the technical level, just more so to the end-user or for discovery it's not as readily apparent.

I agree we don't want to double it up, but here are some concepts I've seen so far:

Popular/Hot (How this is determined is unsure, but I suggest having how this is determined public info wise so people can be aware of how (It's caused issues in the past for some places) )

New/Updated (These being a part of the same category would be fine honestly, new maps and updated maps will generally share the same queue for discovery in a ways)

If the toggles were between those 2 I can see those being highly beneficial.

Another thing to consider too is that instead of recently Modified, but unsure how feasible this is technical wise. But allow worlds a way to "republish" (i.e They resubmit the world to the Cloud) this also saves on folks having accidental saves making it to public maps they've submitted as well too. And there'd just be an original publish date, and a recently republished date.
